v2/lua/openresty_org/controller.lua

```diff
@@ -55,7 +68,29 @@ function _M.run()
         return ngx.redirect(uri .. "/", 301)
     end
 
-    local m, err = re_match(uri, [[ ^ / ( [a-z]{2} ) / (?: ([-\w]+) \.html )? $ ]], 'jox', nil, match_table)
+    local m = re_match(uri, [[ ^ /yum/ ( cn/ )? ( [a-z-0-9]+ ) /OpenResty.repo $ ]], 'jox', nil, match_table)
+    if m then
+        resp_header["Content-Type"] = "text/plain"
+        if m[1] == 'cn/' then
+            repo_file[2] = 'openresty.org/yum/openresty/openresty/epel-'
+        else
+            repo_file[2] = 'copr-be.cloud.fedoraproject.org/results/openresty/openresty/epel-'
+        end
+
+        local distribution = m[2]
+        -- rhel-RELEASE
+        local from, to = re_find(distribution, [[^rhel-(\d+)$]], "jo", nil, 1)
+        if from then
+            repo_file[3] = sub(distribution, from, to)
+        else
+            repo_file[3] = '$releasever'
+        end
+
```

Comment:
We should call `gen_cache_control_headers()`  here to add response headers for caching.